vector/src/main/java/im/vector/app/features/home/room/detail/TimelineFragment.kt

```diff
@@ -1552,7 +1553,7 @@ class TimelineFragment @Inject constructor(
                     attachmentTypeSelector = AttachmentTypeSelectorView(vectorBaseActivity, vectorBaseActivity.layoutInflater, this@TimelineFragment)
                     attachmentTypeSelector.setAttachmentVisibility(
                             AttachmentTypeSelectorView.Type.LOCATION,
-                            vectorPreferences.isLocationSharingEnabled()
+                            BuildConfig.enableLocationSharing
```

Comment:
if we use the `VectorFeatures` with the default value set to `BuildConfig.enableLocationSharing` then we'll also be able to enable/disable location sharing without recompiling the app